crates/backend/src/ast.rs

```diff
@@ -410,6 +411,24 @@ impl Program {
                 ty: class.clone(),
                 kind: MethodKind::Normal,
             }
+        } else if let Some(cls) = wasm.opts.static_method_of() {
+            let class = cls.to_string();
+            let kind = MethodKind::Static;
+
+            let segments = syn::punctuated::Punctuated::from_iter(Some(syn::PathSegment {
+                ident: cls.clone(),
+                arguments: syn::PathArguments::None,
+            }));
+
+            let ty = syn::Type::Path(syn::TypePath {
+                qself: None,
+                path: syn::Path {
+                    leading_colon: None,
+                    segments,
+                },
+            });
```

Comment:
This logic largely redundant with `wasm_bindgen_webidl::util::ident_ty`, it might be good to move some of the utility functions out of that crate and into the backend.